portiaui/app/components/spider-message.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+const { computed } = Ember;
+
+export default Ember.Component.extend({
+    api: Ember.inject.service(),
+    notificationManager: Ember.inject.service(),
+    hasSpider: computed.bool('currentSpider'),
+
+    actions: {
+        runSpider(spider) {
+            this.get('api').post('schedule', {
```

Comment:
That api service is an external library, `ember-api-requests`. We could create our own api service that uses this but so far it is only used here.